<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup</class>
 <widget class="QDialog" name="setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задайте цвета</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>281</width>
     <height>255</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_front">
    <attribute name="title">
     <string>Front</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_right">
    <attribute name="title">
     <string>Right</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_back">
    <attribute name="title">
     <string>Back</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_left">
    <attribute name="title">
     <string>Left</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_up">
    <attribute name="title">
     <string>Up</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_down">
    <attribute name="title">
     <string>Down</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="but_ok">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_cancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Нажмите на кнопку, чтобы изменить цвет</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
